src/main/java/frc/robot/subsystems/vision/Limelight.java

```diff
@@ -0,0 +1,120 @@
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.net.PortForwarder;
+import edu.wpi.first.networktables.*;
+import frc.robot.subsystems.LoggedSubsystem;
+
+import java.util.Optional;
+import java.util.OptionalDouble;
+
+public class Limelight extends LoggedSubsystem<LimelightLogInputs> {
```

Comment:
If I remember correctly, sometimes using networktables, as they add latency, it's more preferable to store the values inside an object, as by the time you pass the check the value of `hasTarget()` can change. (That's why photonvision stores the values in `PhotonPipelineResult`)